buildSrc/src/main/java/Dependencies.kt

```diff
@@ -56,7 +58,7 @@ object Dependencies {
     const val support_constraintlayout = "com.android.support.constraint:constraint-layout:${Versions.constraint_layout}"
     const val support_compat = "com.android.support:support-compat:${Versions.support_libraries}"
 
-    const val arch_lifecycle = "android.arch.lifecycle:extensions:${Versions.lifecycle}"
+    const val arch_lifecycle = "androidx.lifecycle:lifecycle-extensions:${Versions.lifecycle}"
```

Comment:
This is definitely something we won't be able to push at this time: When migrating this to AndroidX then we will have to migrate all other dependencies to AndroidX too. And this will force all apps to migrate too. This is too aggressive and we need to get existing apps to migrate first before we can start using AndroidX in our existing components (No problem for glean though!). #842